packages/web-styles/src/themes/bootstrap/_overrides-variables.scss

```diff
@@ -1,6 +1,7 @@
 @forward '../../variables/animation';
 @forward '../../variables/breakpoints';
 @forward '../../variables/color';
+@forward '../../variables/features';
```

Comment:
Can you move this file to the themes/bootstrap folder as they are bootstrap only features?

Reply:
The file is now located under `themes/bootstrap/_overrides-features.scss`.